src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/container-instrumentation-infrastructure-monitoring-logs.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Container logs configuration for infrastructure monitoring
+tags:
+  - Infrastructure
+  - Install the infrastructure agent
+  - Linux installation
+metaDescription: New Relic's infrastructure monitoring agent automatically instruments container runtimes to collect container metrics and metadata.
+freshnessValidatedDate: never
+---
+
+import infrastructureAddNamesLogs from 'images/infrastructure_screenshot-crop_add-names-logs.webp'
+
+import infrastructureAddContainersLogPath from 'images/infrastructure_screenshot-crop_add-containers-log-path.webp'
+
+import logsParsingUi from 'images/infrastructure_screenshot-full_logs-parsing-ui.webp'
+
+Our [infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring) can gather logs from your containers. When you enable logs from the infrastructure agent, the agent will automatically assign those logs to the hosts running Docker Compose.
+
+## Requirements [#requirements]
```

Comment:
Do customers need to have instrumented their containers? Should we instead link them to the other containers dock as a prerequisite? I imagine they can't collect logs about containers w/o having instrumentation first. 

@harrykimpel

Reply:
@akristen yes, that is correct. I guess it makes sense tzo link them to the other containers doc.